### PR TITLE
commit_msg

### PR DESCRIPTION
**Pull request by tlively**

### CODE REVIEW

**Comment by tlively** [comment]
Current dependencies on/for this PR:
* **#6368** <a href="https://app.graphite.dev/github/pr/WebAssembly/binaryen/6368?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#6367** <a href="https://app.graphite.dev/github/pr/WebAssembly/binaryen/6367?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/WebAssembly/binaryen/6367?utm_source=stack-comment).

**Comment by kripken** [comment]
This and #6368 have no title nor description (aside from "commit_msg"). Maybe some tool breakage?

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 # Binaryen
 
+edit
+
```
Comment (kripken) [inline README.md:6]: The diff also looks fishy?

**Comment by tlively** [comment]
Sorry, I was showing off my workflow in production :)

**Pull request closed without merging**